devito/ir/support/basic.py

```diff
@@ -1330,6 +1326,10 @@ def is_regular(self):
 
 # *** Utils
 
+def vinf(entries):
```

Comment:
nitpick, would do vec_inf

Reply:
but we have vmax, vmin already, so for homogeneity vinf stands better no? we would have to change them all otherwise

Reply:
hm sorry did not pay attention to that